### DISCUSSION TITLE
Document Utilities

### DISCUSSION THREAD

**Discussion by Duke-M-commits** [ReactiveDrop/reactivedrop_translations#106] [Ideas]
It would be cool if somebody could create a wiki-page in which the tools in the utils folder are described in short.
I'm wondering what are they for.

**Comment by BenLubar** [comment]
There's only one script in there and it doesn't accept any arguments.

```
cd utils
go run .
```

It synchronizes the translation files (this gets run automatically by GitHub Actions).

**Reply by Duke-M-commits** [reply]
So the Docker container and the Powershell script are all part of this?
And this one script which makes this percentage of translations too?

**Reply by BenLubar** [reply]
Ah, the Docker container isn't used for anything. The PowerShell script is for syncing one file in reactivedrop_translations with reactivedrop_public_src.
